iml-agent/src/action_plugins/stratagem/action_cloudsync.rs

```diff
@@ -0,0 +1,127 @@
+// Copyright (c) 2020 DDN. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+use crate::{agent_error::ImlAgentError, lustre::search_rootpath};
+use futures::{future::try_join_all, stream, StreamExt, TryStreamExt};
+use iml_wire_types::{FidError, FidItem};
+use liblustreapi::LlapiFid;
+use structopt::clap::arg_enum;
+use tokio::process::Command;
+
+fn get_unique(input: &str) -> String {
+    input.replace('/', "_")
```

Comment:
can't this lead to name collisions?

dir1/foo_bar/file
dir1/foo/bar_file
dir1/foo/bar/file

Possible workaround is how systemd does path -> service names:
'-' -> '--'
'/' -> '-'

Reply:
Done